packages/run-protocol/src/vaultFactory/addAssetToVault.js

```diff
@@ -13,7 +13,13 @@ const reserveThenGetNames = async (nameAdmin, names) => {
 
 /**
  * @param { EconomyBootstrapPowers } powers
- * @param {*} config
+ * @param {object} config
```

Comment:
Agree with the principle that `run-protocol/src` contains logic for the contracts as they run, not how they're deployed. The value in keeping them separated is both for security review and for developers looking to this source for writing contracts and re-using these.

Thankfully this is used only for `scripts/add-collateral-core.js` so it would be easy as moving it over there. Longer term would it make sense to have another package (`@agoric/inter-protocol-vats`?) for the vat setup concern?